component/vshn_postgres.jsonnet

```
@@ -901,6 +901,24 @@ local prometheusRule = {
                   },
                 ],
               },
+              {
+                name: 'postgresql-connections',
+                rules: [
+                  {
+                    alert: 'PostgreSQLConnectionsCritical',
+                    annotations: {
+                      description: 'The connections to {{ $labels.pod }} has been over 90 for 2 hours.\n  Please reducde the load of this instance.',
```

```suggestion
                      description: 'The connections to {{ $labels.pod }} have been over the limit of 90 for 2 hours.\n  Please reduce the load of this instance.',
```
A small type and added a small change.